.github/workflows/main.yml

```diff
@@ -402,13 +402,13 @@ jobs:
       - uses: Swatinem/rust-cache@v2
       - run: cargo fmt -- --check
 
-  semver-checks:
-    runs-on: ubuntu-latest
-    steps:
-      - name: Checkout
-        uses: actions/checkout@v5
-      - name: Check semver
-        uses: obi1kenobi/cargo-semver-checks-action@v2
+  #semver-checks:
+  #  runs-on: ubuntu-latest
+  #  steps:
+  #    - name: Checkout
+  #      uses: actions/checkout@v5
+  #    - name: Check semver
+  #      uses: obi1kenobi/cargo-semver-checks-action@v2
```

Comment:
Nit: I think we could use `continue-on-error: true` instead of commenting out, that way, one can still inspect the error.